lib/menu.rb

```diff
@@ -0,0 +1,32 @@
+
+class Menu
+    
+  MENU = { 
+    'Chicken burger' => 10,
+    'Beef burger' => 12,
+    'Fish dog' => 11,
+    'Hallumi fritter burger' => 8,
+    'Sloppy Joe dog' => 9,
+    'Truffle fries' => 4,
+    'Mac & Cheese' => 5,
+    'Corn ribs' => 4 
+  }
+  
+  attr_reader :menu
+
+  def initialize(menu)
+    @menu = menu
+    list_dishes
+  end
+
+  def list_dishes
+    @menu = MENU if @menu == nil
```

Comment:
I don't really understand why we are requiring a menu parameter with a default of NIL as well as having a constant menu that overwrites this. Was this one of the requirements that you could put in a different menu on initialising? If so it might be worth seeing if there is a way to simplify this.